packages/thirdweb/src/wallets/embedded/core/authentication/error.ts

```diff
@@ -0,0 +1,36 @@
+export const USER_ABORT_AUTHENTICATION_ERROR_MESSAGE =
+  "User aborted the authentication flow";
+
+/**
+ * Error thrown when user aborts the authentication flow
+ */
+export class UserAbortError extends Error {
```

Comment:
these errors are nice but custom errors also bloat bundle size (so let's not over-do it)